app/src/main/java/com/nextcloud/client/database/NextcloudDatabase.kt

```diff
@@ -57,7 +57,7 @@ import com.owncloud.android.db.ProviderMeta
     ],
     version = ProviderMeta.DB_VERSION,
     autoMigrations = [
-        AutoMigration(from = 65, to = 66)
+        AutoMigration(from = 65, to = 67)
```

Comment:
@AlvaroBrey is this needed?
I also changed it in https://github.com/nextcloud/android/pull/8643/files#diff-ffa8f9b74f7af219ebf0927bb94e6c8d03e2b34eb9e39b1287edb1b65532a72fR38

Reply:
If needed, can we combine this? Otherwise I fear we will forget it at some point.

Reply:
```suggestion
        AutoMigration(from = 65, to = 66),
        AutoMigration(from = 66, to = 67),
```

See https://developer.android.com/training/data-storage/room/migrating-db-versions#automated. Your original change would not allow migration from 66 to 67, only from 65.

> If needed, can we combine this? Otherwise I fear we will forget it at some point.

No, we can't combine this. The migration specs need to reference static DB versions as they must be immutable in time, not a constant that will be changed later.